frontend/src/components/ShoppingList.tsx

```diff
@@ -41,17 +39,66 @@ export default function ShoppingList() {
         return <p>Loading...</p>
     }
 
+    function deleteList(id:string) {
+
+        axios.delete("/api/shop/" + id)
+            .then(response => console.log(response.data))
+            .then(fetchLists)
+            .catch(error => console.log(error))
+    }
+
+    const [inputValue, setInputValue] = useState<string>('');
+    const [editingListId, setEditingListId] = useState<string | null>(null);
```

Comment:
Kriege hier folgende Fehlermeldung:
ESLint: React Hook "useState" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?(react-hooks/rules-of-hooks)

Du müsstest die useState nach oben verschieben, vor der IF Condition.

Reply:
die useState wurden auf den Anfang verschoben. Done